apiserver/uniter/uniter.go

```diff
@@ -1564,8 +1581,10 @@ func (u *UniterAPIV3) AddMetricBatches(args params.MetricBatchParams) (params.Er
 	return result, nil
 }
 
+// TODO(wpk) Uniter.NetworkConfig API is obsoleted by Uniter.NetworkInfo
 // NetworkConfig returns information about all given relation/unit pairs,
 // including their id, key and the local endpoint.
+// It's not included in APIv6
 func (u *UniterAPIV3) NetworkConfig(args params.UnitsNetworkConfig) (params.UnitNetworkConfigResults, error) {
```

Comment:
We care and it's there:
// Mask it from old version
func (u *UniterAPIV3) NetworkInfo(_, _ struct{}) {}